backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/handlingunits/process/WEBUI_M_ReceiptSchedule_ReceiveHUs_UsingDefaults.java

```diff
@@ -129,6 +128,16 @@ private void adjustLUTUConfiguration(final I_M_HU_LUTU_Configuration lutuConfig,
 					lutuConfig.setQtyTU(qtyToMoveTU);
 				}
 			}
+
+			// Adjust CU if TU can hold infinite
+			if (lutuConfig.isInfiniteQtyCU())
+			{
+				lutuConfig.setIsInfiniteQtyCU(false);
+				// This is similar to the default behaviour of limited capacity packaging
```

Comment:
pls write a short comment about why we do this

"this is similar..." ... it's not an actual argument.

Reply:
added short info in comments.